### PR TITLE
Feature/create returns

### PR DESCRIPTION
**Pull request by jmassie**
Added a new returns table, post and get api using database transactions and postman tests. Issue: Scottish-Natural-Heritage/Licensing#762

### CODE REVIEW

File: src/controllers/returns.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ */
+const ReturnsController = {
+  /**
+   * Create a new randomly allocated Return wrapped in a database transaction.
+   *
+   * Transaction completes all requests and returns the new return transaction id.
+   *
+   * @param {number} id An existing return's ID.
+   * @param {any} cleanObject A new revocation object to be added to the database.
+   * @returns {number} The new Return transaction id.
```
Comment (MikeCoats) [inline src/controllers/returns.js:16]: Technically this is `{number | undefined}`.

File: src/models/returns.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      },
+      fromDate: {
+        type: Sequelize.DATE
+      },
+      toDate: {
+        type: Sequelize.DATE
+      },
+      comment: {
+        type: Sequelize.TEXT
+      },
```
Comment (MikeCoats) [inline src/models/returns.js:39]: Are all these fields 'nullable', or should some of them be mandatory?

File: util/db/migrations/20210729104712-sfo-recreate-returns.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      },
+      fromDate: {
+        type: Sequelize.DATE
+      },
+      toDate: {
+        type: Sequelize.DATE
+      },
+      comment: {
+        type: Sequelize.TEXT
+      },
```
Comment (MikeCoats) [inline util/db/migrations/20210729104712-sfo-recreate-returns.js:32]: The same question, about possible 'nullable' fields, here too.

**Review by MikeCoats — CHANGES_REQUESTED** [review]
One bit of documentation needs updating and there's two questions about the nullability of some of the fields.

File: src/controllers/returns.js (hunk 1 above)
Comment (JohnJAMoore) [reply src/controllers/returns.js]: resolved https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/30/commits/acdef55b0a4c2c8d702acb2c868b79fb148ea555

File: src/models/returns.js (hunk 2 above)
Comment (JohnJAMoore) [reply src/models/returns.js]: resolved https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/30/files/0e442ea6c243eb5cf653655b019f9dfed092f0b6#diff-db980472dfb81403f7a047517b63b7987831d84c00883d6bcb68f1064fe2b295

File: util/db/migrations/20210729104712-sfo-recreate-returns.js (hunk 3 above)
Comment (JohnJAMoore) [reply util/db/migrations/20210729104712-sfo-recreate-returns.js]: resolved https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/30/commits/d0befc3abf7f62d8acfa1d2233904ffdb86f9288

**Review by MikeCoats — APPROVED** [review]

**Pull request merged**